### PR TITLE
Bug 1874510: vendor: bump kube deps to 1.19

### PR DESCRIPTION
**Pull request by runcom**
bumping k8s deps to align with what we have cluster wide.

Signed-off-by: Antonio Murdaca <runcom@linux.com>

### CODE REVIEW

**Comment by runcom** [comment]
> Job execution failed: Pod pending timeout.

/retest

File: go.mod [hunk 1]
```diff
@@ -27,73 +27,70 @@ require (
...
 	github.com/hashicorp/golang-lru v0.5.3 // indirect
 	github.com/huandu/xstrings v1.2.0 // indirect
 	github.com/imdario/mergo v0.3.9
 	github.com/magiconair/properties v1.8.1 // indirect
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/openshift/api v3.9.1-0.20191111211345-a27ff30ebf09+incompatible
-	github.com/openshift/client-go v0.0.0-20200320150128-a906f3d8e723
-	github.com/openshift/library-go v0.0.0-20200320155611-2a351bebf158
-	github.com/openshift/runtime-utils v0.0.0-20191011150825-9169de69ebf6
+	github.com/openshift/api v0.0.0-20200829102639-8a3a835f1acf
```
Comment (ashcrow) [inline go.mod:41]: Is the version downgrade due to the update not being officially released yet?

File: go.mod [hunk 2]
```diff
@@ -27,73 +27,70 @@ require (
...
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
-	google.golang.org/appengine v1.6.1 // indirect
-	k8s.io/api v0.18.3
-	k8s.io/apiextensions-apiserver v0.18.0
-	k8s.io/apimachinery v0.18.3
-	k8s.io/client-go v0.18.0
-	k8s.io/code-generator v0.18.3
-	k8s.io/kubectl v0.0.0
-	k8s.io/kubelet v0.18.0
+	k8s.io/api v0.19.0
```
Comment (ashcrow) [inline go.mod:56]: k8s items look correct based on the PR text.

File: go.mod (hunk 1 above)
Comment (runcom) [reply go.mod:41]: the versions are misleading, this is newer than the previous one 20191111211345 vs 20200829102639

**Comment by ashcrow** [comment]
Neither are required but let's see if they pass

/test okd-e2e-aws
/test e2e-ovn-step-registry

**Comment by kikisdeliveryservice** [comment]
Both of those tests are permafailing..

/skip

**Comment by runcom** [comment]
@kikisdeliveryservice `/lGtM`

**Pull request merged**